classes/notification/managerDelegate/PendingRevisionsNotificationManager.inc.php

```diff
@@ -97,6 +97,7 @@ public function updateNotification($request, $userIds, $assocType, $assocId) {
 		$userId = current($userIds);
 		$submissionId = $assocId;
 		$stageData = $this->_getStageDataByType();
+		if ($stageData == null) return;
```

Comment:
I see a few conditions added here relating to `$stageData` not being present. What's the use case for that? I think the stage data should reliably describe the application, no? (I don't know this aspect of the codebase well.)